client/extensions/woocommerce/app/settings/payments/stripe/payment-method-stripe-placeholder-dialog.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Dialog from 'components/dialog';
+
+class PaymentMethodStripePlaceholderDialog extends Component {
```

Comment:
Maybe stateless functional component?

Reply:
Excellent idea - done in https://github.com/Automattic/wp-calypso/pull/19115/commits/bf9118c3610fb69e4b976be39bdada397f526d1e